README.md

```diff
@@ -212,6 +212,7 @@ The following is a list of static resources.
 - [azure_web_app_functions](docs/resources/azure_web_app_functions.md)
 - [azure_webapp](docs/resources/azure_webapp.md)
 - [azure_webapps](docs/resources/azure_webapps.md)
+- [azure_express_route_providers](docs/resources/azure_express_route_providers.md)
```

Comment:
Can you put this in alphabetical order with the others.

Reply:
Yes Ian.